crates/iota-indexer/src/store/pg_indexer_store.rs

```diff
@@ -722,6 +722,38 @@ impl PgIndexerStore {
         })
     }
 
+    fn persist_tx_insertion_order_chunk(
+        &self,
+        tx_order: Vec<NewTxInsertionOrder>,
+    ) -> Result<(), IndexerError> {
+        let guard = self
+            .metrics
+            .checkpoint_db_commit_latency_tx_insertion_order_chunks
+            .start_timer();
+
+        transactional_blocking_with_retry!(
+            &self.blocking_cp,
+            |conn| {
+                for tx_order_chunk in tx_order.chunks(PG_COMMIT_CHUNK_SIZE_INTRA_DB_TX) {
+                    insert_or_ignore_into!(tx_insertion_order::table, tx_order_chunk, conn);
+                }
+                Ok::<(), IndexerError>(())
+            },
+            PG_DB_COMMIT_SLEEP_DURATION
+        )
+        .tap_ok(|_| {
+            let elapsed = guard.stop_and_record();
+            info!(
+                elapsed,
+                "Persisted {} chunked txs insertion order",
+                tx_order.len()
+            );
+        })
+        .tap_err(|e| {
+            tracing::error!("Failed to persist txs insertion order with error: {}", e);
```

Comment:
```suggestion
            tracing::error!("Failed to persist txs insertion order with error: {e}");
```

Reply:
There are some more cases like this in this PR

Reply:
Done 80f6e8144984cf601c33ecaee512ae8489e648de